nodes/mtnode.py

```diff
@@ -57,6 +57,7 @@ def __init__(self):
         baudrate = get_param('~baudrate', 0)
         timeout = get_param('~timeout', 0.002)
         initial_wait = get_param('~initial_wait', 0.1)
+        output_config = get_param('~output_config', '')
```

Comment:
Any reason why you don't set the same default as in the launch file here? `oq,ad,wd,mf`

Reply:
no particular reason ... so I added them in, to stick to your original style of default params